_targets.R

```diff
@@ -750,5 +752,27 @@ list(
              get_nested_gages(gagesii = p1_sites_g2,
                               nav_distance_km = nav_distance_km),
              deployment = 'worker'
-            )  
+            )  ,
+  
+  ###EDA plots
+  ##list of all the metrics names - for dynamic branching
+  tar_target(p5_all_metrics_names,
+             c(colnames(p1_HIT_metrics)[-1],colnames(p1_FDC_metrics)[-1])),
+  
+  ##combined metrics tables
+  tar_target(p5_all_metrics,
```

Comment:
We could label this as `p2` instead (which will be the phase for metrics). It addresses #29